src/SignalR/clients/csharp/Client/test/UnitTests/LongPollingTransportTests.cs

```diff
@@ -692,4 +693,45 @@ public async Task SendsDeleteRequestWhenTransportCompleted()
             Assert.Equal(TestUri, deleteRequest.RequestUri);
         }
     }
+
+    [Fact]
+    public async Task StartAsyncSetsAcceptHeaderCorrectly()
+    {
+        var mockHttpHandler = new Mock<HttpMessageHandler>();
```

Comment:
Why use `Mock` for this test and `TestHttpMessageHandler` for the other one? I'd prefer staying away from Mock in general (even though we use it for some tests).

Reply:
In all honestly, I was just sticking as close to the practices/methodologies found within the same testing classes. I have amended all the tests I wrote to use `TestHttpMessageHandler`. Which I should've originally done.